projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx

```diff
@@ -1,187 +1,130 @@
-import {
-    DotCMSContentTypeField,
-    DotCMSDateField,
-    DotCMSKeyValueField,
-    DotCMSMultiSelectField,
-    DotCMSTextFieldField,
-    DotCMSDateRangeField
-} from '../../models';
+import { DotCMSContentTypeField } from '../../models';
 
 export const DotFormFields = {
-    Text: (field: DotCMSTextFieldField) => (
+    Text: (field: DotCMSContentTypeField) => (
```

Comment:
This fields are the one we're using in the `dot-form` which means they're coming from the backend, in the backend we don't have extra attributes like `required-message` or `min` etc... so we don't need to create this models for the form. Our custom elements support more attr that the ones that comes from the backend and that's ok but to build our dot-form we don't need to add them.

Probably in the future the fields we create in backend will have more stuff.